docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+services:
+  db:
+    image: postgres:11.2
+    volumes:
+      - ./tmp/db:/var/lib/postgresql/data
+  redis:
+    image: redis:5.0
```

Comment:
On checking the postgres running on live at the moment is only `4.0.14`, having this set higher locally could make things harder to debug.